app/models/concerns/blacklight/document.rb

```diff
@@ -82,6 +84,11 @@ def first key
     Array(self[key]).first
   end
 
+  def inspect
+    fields = inspector_fields.map { |field| "#{field}: #{public_send(field)}" }.join(", ")
```

Comment:
I wonder if just `:id` is too limiting. I often use `inspect` to see what data we got back from solr (and... I'm not sure I've ever inspected a `SolrDocument` except to do that?). Would it be possible to include all of `_source` by default (and omit `response`, which I 💯  agree is unwieldy and easy to get at when it's useful)?

Reply:
Sure, I changed the default to `_source`